app/src/main/java/edu/uc/group/rankine/ui/main/MainActivity.kt

```diff
@@ -1,9 +1,9 @@
 package edu.uc.group.rankine.ui.main
 
-import android.app.SearchManager
-import android.content.Context
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
+import android.app.SearchManager
+import android.content.Context
 import android.os.Bundle
 import android.view.Menu
 import android.view.MenuItem
```

Comment:
Deleted imports that were not used.

Reply:
Again... looks like they were just re-ordered here.  Same imports exist before as after.